frontend/src/pages/SignupPage/index.js

```diff
@@ -168,13 +191,26 @@ const SignupPage = () => {
   const buttonHandler = state.step === STEP_4 ? submitData : continueToNextStep;
 
   return (
-    <div className={styles.root}>
-      <div className={styles.logo}>
-        <Logo />
-      </div>
-      <div className={styles.main}>{stepInfo[state.step].component}</div>
-      <Button onClick={buttonHandler}>{stepInfo[state.step].buttonText}</Button>
-    </div>
+    <>
+      <MainContainer>
+        <div
```

Comment:
So instead of having a blank `div` here, you can now move the `BackLinkWrapper` styles from the `BackLink` component into here.

I mentioned this previously to @jreis13 so he can maybe explain it if necessary. Basically, I explained to him that layout styles shouldn't be applied to a component like `BackLink` but to the parent.

That way, this `div` would be `BackLinkWrapper` and you'd chain a class `.isHidden` when needed.